lib/simplecov-html/version.rb

```diff
@@ -1,7 +1,7 @@
 module SimpleCov
   module Formatter
     class HTMLFormatter
-      VERSION = "0.10.1"
+      VERSION = "0.10.1".dup
 
       def VERSION.to_a
```

Comment:
Should I change things so VERSION returns an object with these methods, plus `to_s` that returns the full string? Or should VERSION just be a normal string, no monkey-patched methods?